test/resources/oc.resource

```diff
@@ -41,16 +41,29 @@ Oc Exec
     ${output}=    Run With Kubeconfig    oc exec -n ${NAMESPACE} pod/${pod} -- /bin/bash -c '${cmd}'
     RETURN    ${output}
 
-Oc Wait For
+Oc Wait For Object By KindName
     [Documentation]    Runs 'oc wait'.
-    ...    ${type_name}    expects a "type/name", e.g. pod/foo
+    ...    ${kind_name}    expects a concatenated string in the scheme of "kind/name", e.g. pod/foo
     ...    ${wait_for}    expects a key=value. See 'oc wait --help' for examples
     ...    ${timeout}    Takes an integer and a single character time-unit identifier, e.g.: 30s
     ...    If no timeout value is provided, defaults to 30s.
-    [Arguments]    ${type_name}    ${wait_for}    ${timeout}=30s
+    [Arguments]    ${kind_name}    ${wait_for}    ${timeout}=30s
 
     ${timeout_opt}=    Set Variable    --timeout\=${timeout}
-    ${output}=    Run With Kubeconfig    oc wait -n ${NAMESPACE} ${type_name} --for=${wait_for} ${timeout_opt}
+    ${output}=    Run With Kubeconfig    oc wait -n ${NAMESPACE} ${kind_name} --for=${wait_for} ${timeout_opt}
+    RETURN    ${output}
+
+Oc Wait For Object By Label
```

Comment:
This is still named procedurally "wait for..." instead of asserting some state "pod with label is ready".

Someone reading the test code should be able to read it more like a story than a program.